ansible/devspace-runtime.yml

```diff
@@ -0,0 +1,36 @@
+- name: Run devspace
+  hosts: devspace
+  connection: ssh
+  gather_facts: no
+  become_user: "{{ devuser }}"
+
+  environment:
+    USER_ID: "{{ user_id }}"
+
+  tasks:
+    - name: Devspace down
+      docker_service:
+        project_src: "{{ devhome }}"
+        state: absent
```

Comment:
just discussed. Playbook should be idempotent, is better if we have `devspace_force_restart: false`